package.xml

```diff
@@ -3,7 +3,7 @@
   <!-- This is a Catkin package.xml file to optionally support building DART in
        a Catkin workspace. Catkin is not required to build DART. For more
        information, see: http://ros.org/reps/rep-0136.html -->
-  <name>dartsim</name>
+  <name>DART</name>
```

Comment:
IIRC, we used `dartsim` for a reason. Maybe it's already taken by the Dart language?